spring-cloud-dataflow-registry/src/main/java/org/springframework/cloud/dataflow/registry/support/AppResourceCommon.java

```diff
@@ -121,71 +170,20 @@ else if (scheme.equals("docker")) {
 				else {
 					resourceLoader = new DownloadingUrlResourceLoader();
 				}
-				return resourceLoader.getResource(uriString);
+				return resourceLoader.getResource(resourceUri);
```

Comment:
Not a biggie but could you rename `resourceLoader` used locally here as same symbol `resourceLoader` is already used as a class field. Way too easy to mix when you're just eyeballing this section.

Reply:
+1